apps/gateway/test/champions_test.exs

```diff
@@ -295,6 +297,105 @@ defmodule Gateway.Test.Champions do
     end
   end
 
+  describe "items" do
+    test "get item", %{socket_tester: socket_tester} do
+      {:ok, user} = Users.register("GetItemUser")
+
+      {:ok, epic_bow} =
+        Items.insert_item_template(%{
+          game_id: Utils.game_id(),
+          name: "Epic Bow of Testness",
+          type: "weapon"
+        })
+
+      {:ok, item} = Items.insert_item(%{user_id: user.id, template_id: epic_bow.id, level: 1})
+
+      :ok = SocketTester.get_item(socket_tester, user.id, item.id)
+      fetch_last_message(socket_tester)
+
+      %WebSocketResponse{
+        response_type: {:item, %Item{} = fetched_item}
+      } = get_last_message()
+
+      assert fetched_item.id == item.id
+      assert fetched_item.user_id == user.id
+      assert fetched_item.unit_id == ""
```

Comment:
This is perfect but let's add a comment explaining it just in case it's not clear enough for someone not familiar with our app and protobuf.

Something along the lines of:
`# We expect the item to be unequipped after creation. Since protobuf can't handle null messages, we get an empty string.`